app/features/settings/metrics/SettingsMetricsPanel.js

```diff
@@ -0,0 +1,161 @@
+import React, {Component} from 'react';
+import {
+    Text,
+    View,
+    TouchableOpacity,
+    Picker,
+    Platform,
+    Switch,
+    StyleSheet
+} from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome'
+
+import { SETTINGS_PANEL_STYLES } from 'app/appearance/styles/GlobalStyles';
+import * as DateUtils from 'app/utility/transforms/DateUtils';
+import SettingsCollapsedMetrics from './metric/SettingsCollapsedMetrics';
+import SettingsCollapsedAvgMetrics from './metric/SettingsCollapsedAvgMetrics';
+import SettingsCollapsedBestEverMetrics from './metric/SettingsCollapsedBestEverMetrics';
+import SettingsQuantifiers from './quantifier/SettingsQuantifiers';
+
+class SettingsMetricsPanel extends Component {
+
+    // ACTIONS
+
+    _tapcurrentMetricPosition(metricPosition, quantifier) {
+        this.props.tapMetric(metricPosition, quantifier)
+    }
+
+    _tapcurrentQuantifierPosition(metricPosition, quantifierPosition) {
+        this.props.tapQuantifier(metricPosition, quantifierPosition)
+    }
+
+    _renderMetric(metric, metricPosition, quantifier) {
+        return (
+            <TouchableOpacity style={[SETTINGS_PANEL_STYLES.blueButton, {width: 150, height: 30, marginLeft: 10, marginBottom: 10}]} onPress={() => this._tapcurrentMetricPosition(metricPosition, quantifier)}>
+                <Text style={SETTINGS_PANEL_STYLES.buttonText}>
+                    {metric}  <Icon name="caret-down" size={10} color='white' />
+                </Text>
+            </TouchableOpacity>
+        );
+    }
+
+    _renderQuantifier(metricPosition, quantifier, quantifierPosition) {
+        return (
+            <TouchableOpacity style={[SETTINGS_PANEL_STYLES.blueButton, {width: 100, height: 30, marginLeft: 15, marginBottom: 10}]} onPress={() => this._tapcurrentQuantifierPosition(metricPosition, quantifierPosition)}>
+                <Text style={SETTINGS_PANEL_STYLES.buttonText}>
+                    {quantifier}  <Icon name="caret-down" size={10} color='white' />  
+                </Text>
+            </TouchableOpacity>
+        );
+    }
+
+    _renderRow(metric, metricPosition, quantifier, quantifierPosition) {
+        let num = metricPosition.substr(metricPosition.length - 1);
```

Comment:
- [ ] const

- [ ] instead of parsing the metric position, literally just pass in the integer from the render() function itself